submissions/LuckyDnepr/Hooli-popup-task/styles-menu.css

```diff
@@ -0,0 +1,248 @@
+html, body {
+    width: 90%;
+    height: 100%;
+    margin: 0px;
+}
+
+html {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+a {
+    text-decoration: none;
+    font-size: 16px;
+    color: black;
+}
+
+/*Menu button styles start*/
+
+#nav_keeper_input {
```

Comment:
Please see the [video](https://www.youtube.com/watch?v=E6kLaaQFctU&ab_channel=VadimMakeev)